cmd/agent/subcommands/snmp/command.go

```diff
@@ -29,272 +30,267 @@ import (
 )
 
 const (
-	defaultVersion = ""
-	defaultOID     = ""
-	defaultPort    = 161
-
-	// snmp v1 & v2c
-	defaultCommunityString = "public"
-
-	// snmp v3
-	defaultUserName      = ""
-	defaultAuthProtocol  = ""
-	defaultAuthKey       = ""
-	defaultPrivProtocol  = ""
-	defaultPrivKey       = ""
-	defaultContext       = ""
-	defaultSecurityLevel = ""
```

Comment:
I don't have a strong opinion on this but FYI when we first implemented the command we decided to have these variables even though they all have the same value of an empty string to make it clearer what they represent later on

Reply:
I dropped them because A) I worried it made it harder to read the lines that were actually setting up the flags, and B) I thought the names were confusing for things like `defaultVersion` and `defaultSecurityLevel`, where the *flags* default to `""` but the actual value that gets used doesn't, because of additional logic that guesses defaults from other arguments. If anyone feels strongly about it I can put these back.